test/mocks/mocks.go

```diff
@@ -21,6 +21,8 @@ import (
 
 // resource details employed in mocks
 const (
+	// Fixme(Akash): This values are tightly coupled with postgresql operator.
+	// Need to make it more dynamic.
```

Comment:
I think we have those values as constants in `mocks.go`, if not, would be good to organize in the same approach than in `common.go` (under controller).